controllers/users.js

```diff
@@ -711,6 +712,24 @@ async function usersPatchHandler(req, res) {
   }
 }
 
+const addGithubCreatedAtKey = async (req, res) => {
+  try {
+    const users = await userQuery.fetchUsersWithoutGithubCreatedAtKey();
+    if (!users.length) {
+      return res.status(404).json({ message: "No users found without github created at key!" });
+    }
+    const usersGithubKey = await githubModalCircularDependency(users);
+    await userQuery.addGithubCreatedAtKey(usersGithubKey);
+
+    return res.status(200).json({
+      message: "Github created at added for all users!",
+      usersFound: users.length,
+    });
```

Comment:
If possible we should also return users updated also here?

Reply:
done